docs/_data/nakadi-event-bus-api.yaml

```
@@ -2624,6 +2624,8 @@ definitions:
       number of clients by managing consumed event offsets and distributing load between instances.
       The key properties that identify subscription are 'owning_application', 'event_types' and 'consumer_group'.
       It's not possible to have two different subscriptions with these properties being the same.
+      Trying to create a subscription with these keys matching an existing one will just return that one, and
```

maybe it is a bit clearer that way 
```suggestion
      Creating a subscription with the keys matching existing subscription will neither create a new or change the existing subscription ignoring the remaining fields (e.g. `read_from`, `authorization`, etc.).
```

---

Thanks, I've applied this (with a tiny bit of grammar fix).